sources/packages/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -618,4 +620,43 @@ export class ApplicationStudentsController extends BaseController {
       }
     }
   }
+
+  /**
+   * Get application to request appeal.
+   ** Application eligible to be requested for
+   ** a change will be returned.
+   * @param applicationNumber
+   * @param userToken
+   * @returns application
+   */
+  @ApiOkResponse({
+    description: "Returns application which can be requested for change.",
+  })
+  @ApiNotFoundResponse({
+    description:
+      "Application either not found or not eligible to request for change.",
+  })
+  @Get(":applicationNumber/appeal")
+  async getApplicationToRequestChange(
+    @Param("applicationNumber") applicationNumber: string,
+    @UserToken() userToken: IUserToken,
+  ): Promise<ApplicationPrimaryDTO> {
+    const application =
+      await this.applicationService.getApplicationToRequestChange(
```

Comment:
I would say that internally we are calling "Request a Change" appeal hence I would suggest renaming the method to `getApplicationToRequestAppeal`, or something similar.